crates/sequencer/src/rpc.rs

```diff
@@ -117,28 +119,21 @@ pub(crate) fn create_rpc_module<C: sov_modules_api::Context>(
 
             debug!("Sequencer: citrea_sendRawDepositTransaction");
 
-            let evm = Evm::<C>::default();
-            let mut working_set = WorkingSet::<C>::new(ctx.storage.clone());
-
-            let dep_tx = ctx
-                .deposit_mempool
-                .lock()
-                .await
-                .make_deposit_tx_from_data(deposit.clone().into());
-
-            let tx_res = evm.get_call(dep_tx, None, None, None, &mut working_set);
```

Comment:
Since there is a nice way to parse and validate deposit params - no need to make a virtual evm call to check it.

Reply:
the evm call was not to check params, it was a way for us to verify the deposit. we must have it here. pls add it back